### ISSUE TITLE
Email Domain Is Not Accepted for Registration Confirmation

### ISSUE THREAD

**Issue by awhisler** [HTBox/crisischeckin#336]
**Steps:**
Register
Fill out required fields
Enter rare/personal email domain (ali.whisler@lostranch.com)
Create new account

**Expected Results:**
Should send email confirmation

**Actual Results:**
Email does not get sent to confirm registration

**Comment by jsucupira** [comment]
I can look into this.

**Comment by wadewachs** [comment]
mailinator.com was another email service that was having problems receiving registration emails. I was able to get mailinator.com to receive email when an administrator sends a message to the volunteers.

**Comment by jsucupira** [comment]
I misunderstood the original issue.  I originally thought it was an issue with email validation.
Looks like this is a spam filtering issue? if so I rather not work on this, since this is configuration problem vs a coding problem.

**Comment by awhisler** [comment]
I don't believe it is a spam filtering issue. It is not in my spam folder and I still haven't received the registration confirmation email. 

Thanks, AliSent from my Samsung device

-------- Original message --------
